config/oauth2_config.xml.sample

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
Could this be merged in with `config/auth_conf.xml` with `<type>oauth2</type>` or something similar?

Reply:
@erasche Not sure `config/auth_conf.xml.sample` is a good fit, because it may contain multiple auth mechanisms which are tried in sequence (unless `<continue-on-failure>False</continue-on-failure>` is specified for a matched authenticator) using the provided username and password.

Reply:
@nsoranzo good point, I had forgotten that those were tried serially with credentials. Just didn't want to manage an extra file if it was a possibility.